aep/general/0002/aep.md

```diff
@@ -0,0 +1,64 @@
+# AEP Numbering and Organization
+
+This document describes the numbering system and general organization of AEPs.
+
+AEP numbers:
+
+- Are unique. No two AEPs share the same number.
+- Indicate the order in which they are authored.
+
+To help with organization and discovery, AEPs also have a category and a slug
+to easily discovery and refer to the AEP.
+
+## AEP Numbers
+
+### Core AEPs
+
+To uniquely identify an AEP, each one is assigned a permanent number once it
```

Comment:
Nit:  the number has to be assigned before merging, right?

Reply:
how about "as it's merged"?

Reply:
updated to use "as it's merged" language, PTAL.